pkg/sync/filepath_sync.go

```diff
@@ -14,6 +17,9 @@ type FilePathSync struct {
 	URI          string
 	Logger       *logger.Logger
 	ProviderArgs ProviderArgs
+	// FileType indicates the file type e.g., json, yaml/yml etc.,
+	// TODO: limit this to `json` or `yaml only?`
```

Comment:
Please address this TODO.

Reply:
After a second thought, I removed the TODO because we already have a switch case to check the file type.

Reply:
Please unresolve this conversation if you have any feedback for me.